reporting/CassandraBackend.cpp

```diff
@@ -1556,80 +1575,73 @@ CassandraBackend::open(bool readOnly)
         if (!upperBound2_.prepareStatement(query, session_.get()))
             continue;
     */
-        query = {};
+        query.str("");
         query << "SELECT TOKEN(key) FROM " << tablePrefix << "objects "
               << " WHERE key = ? LIMIT 1";
 
         if (!getToken_.prepareStatement(query, session_.get()))
             continue;
 
-        query = {};
-        query << "SELECT key FROM " << tablePrefix << "books "
-              << " WHERE book = ? AND sequence <= ? AND deleted_at > ? AND"
-                 " key > ? "
-                 " ORDER BY key ASC LIMIT ? ALLOW FILTERING";
-        if (!getBook_.prepareStatement(query, session_.get()))
-            continue;
-        query = {};
-        query << "SELECT key FROM " << tablePrefix << "books2 "
-              << " WHERE book = ? AND sequence = ? AND "
-                 " key > ? "
-                 " ORDER BY key ASC LIMIT ?";
+        query.str("");
+        query << "SELECT quality_key FROM " << tablePrefix << "books2 "
```

Comment:
Let's change everything that is called some form of `books2` to just `books`, and remove the old `books` stuff.